### DISCUSSION TITLE
Hotkeys for fast launch browser

### DISCUSSION THREAD

**Discussion by vnaumenko** [U-C-S/Hurl#63] [Ideas]
**Hello, thanks for app.**

It would be great to be able to launch the browser by pressing the selected key.

_For example:_ 
1. Clicked on the link
2. App opens with three browsers
3. Press "2"
4. The second browser launches

The key can be set in the settings.

**Comment by U-C-S** [comment]
yeh Its in the milestone for next update. I also have a prototype locally, might commit that soon for a v0.8 alpha release
